src/component/parser/json/converter/DiagramConverter.ts

```diff
@@ -80,8 +80,7 @@ export default class DiagramConverter {
 
     shapes = ensureIsArray(shapes);
 
-    for (let i = 0; i < shapes.length; i++) {
-      const shape = shapes[i];
+    for (const shape of shapes) {
```

Comment:
I suggest we use `ensureIsArray(shapes).forEach` directly for consistency with the code of this class